dll/win32/shell32/wine/pidl.c

```diff
@@ -527,6 +528,21 @@ BOOL _ILHACKCompareSimpleIds(LPCITEMIDLIST pidltemp1, LPCITEMIDLIST pidltemp2)
 
     return TRUE;
 }
+#else
+BOOL _ILCompareSimpleIDs(LPCITEMIDLIST pidl1, LPCITEMIDLIST pidl2)
```

Comment:
This function can be rewritten to be a single return expression.

Reply:
It could, but that does not make it better or more readable.

Reply:
> not [...] more readable

I have a different point of view:
```
if (false cond) return false
else if (false cond) return false
return true
```
looks quite bloated to me, even more with the additional braces.

Reply:
Eventually improved/done.